### DISCUSSION TITLE
Seperate syntaxes/themes as a standalone repository

### DISCUSSION THREAD

**Discussion by Freed-Wu** [sharkdp/bat#2371] [Ideas]
How about separating syntaxes/themes as a standalone repository? Because not only bat user but also sublime user maybe need it, and they don't want to clone the whole repository of bat. Just my 2c, TIA!

**Comment by keith-hall** [comment]
There was a discussion regarding this in an issue, before GitHub discussions were a thing: https://github.com/sharkdp/bat/issues/919
